backend/src/packages/users/user.model.ts

```diff
@@ -31,11 +31,11 @@ class UserModel extends AbstractModel {
     return {
       withoutPassword(builder): QueryBuilder<UserModel> {
         return builder.select(
-          'details.id',
+          'users.id',
           'email',
           'details.fullName',
-          'details.created_at',
-          'details.updated_at',
+          'users.created_at',
+          'users.updated_at',
```

Comment:
Let's add users even if fields don't conflict, and also we don't need to specify details fields
```suggestion
          'users.id',
          'users.email',
          'users.created_at',
          'users.updated_at',
```

Reply:
But we could also do smth like this instead of modifier
      .modify((builder) => {
        void builder.where('users.id', id).select('users.id', 'users.email', 'users.createdAt', 'users.updatedAt', 'details.fullName').first();
      })

Reply:
I changed to users, checked that its returning data properly